dicomImport/matRad_interpDoseCube.m

```diff
@@ -0,0 +1,114 @@
+function [ dose ] = matRad_interpDoseCube( ct, currDose )
```

Comment:
maybe we should also rename this functiont to matRad_interpDicomDoseCube...

Otherwise people could think it may be useful to generall resample any matRad dose cube. which is not true... BTW This also applies to the function matRad_interpCtCube.